### PR TITLE
Mobile Styles for sign in page

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/94?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #80** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/80?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #81** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #83** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/83?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #84** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/84?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #87** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/87?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #92** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/92?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #93** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/93?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #94** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/94?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/94?utm_source=stack-comment).

File: packages/app/components/sidebar/SignInSideBar.tsx [hunk 1]
```diff
@@ -29,16 +29,16 @@ const SignInSchema = z.object({
 })
 
 const SignInSideBar = ({ ...props }: YStackProps) => (
-  <SideBar width="28%" maw="405px" {...props}>
-    <Stack href={'/'} f={1} jc="center">
-      <IconSendLogo />
+  <SideBar width="28%" minWidth="3px" maw="405px" px="$4" {...props}>
```
Comment (0xBigBoss) [inline packages/app/components/sidebar/SignInSideBar.tsx:32]: Do pixel values work on native? I thought it just takes a number.

File: packages/app/features/auth/sign-in-screen.tsx [hunk 2]
```diff
@@ -1,67 +1,121 @@
...
 
 export const SignInScreen = () => {
   return (
     <Container>
-      <Stack pos="relative" h="100svh" f={1} ai="center" jc="center">
-        <View pos="absolute" top={0} left={0} mt="auto" mb="auto" zIndex={-1} w="100%" h="100%">
-          <SignInCarousel />
-        </View>
-      </Stack>
+      <YStack w="100%" h="95svh" jc="space-around" mb="auto" mt="auto">
```
Comment (0xBigBoss) [inline packages/app/features/auth/sign-in-screen.tsx:23]: I don't think the `95svh` will work on native, will it?

File: packages/app/features/auth/sign-in-screen.tsx [hunk 3]
```diff
@@ -72,33 +126,80 @@ const SignInCarousel = () => {
...
-            right={0}
-            btw="273px"
-            brw="90px"
-          />
-          <View position="absolute" bottom={'$0'} right={'$0'}>
-            <IconSLogo size={'$4'} />
-          </View>
-          {image}
-        </View>
+        <>
```
Comment (0xBigBoss) [inline packages/app/features/auth/sign-in-screen.tsx:140]: Doesn't need a key prop?

**Review by 0xBigBoss — APPROVED** [review]
LGTM

**Review by 0xBigBoss — APPROVED** [review]
LGTM

**Pull request merged**